google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardFacetEditor.java

```diff
@@ -166,50 +167,42 @@ private boolean isManagedLibrariesEnabled() {
 
     @Override
     public void afterLibraryAdded(final Library addedLibrary) {
-      AppEngineStandardMavenLibrary library = AppEngineStandardMavenLibrary
-          .getLibraryByMavenDisplayName(addedLibrary.getName());
-
-      if (library == null) {
-        return;
-      }
-
-      final DependencyScope scope = library.getScope();
-
-      new WriteAction() {
-        @Override
-        protected void run(@NotNull Result result) throws Throwable {
-          ModuleRootManager manager = ModuleRootManager.getInstance(context.getModule());
-          ModifiableRootModel model = manager.getModifiableModel();
-
-          model.addLibraryEntry(addedLibrary);
-
-          for (OrderEntry orderEntry : model.getOrderEntries()) {
-            if (orderEntry.getPresentableName().equals(addedLibrary.getName())) {
-              ((ExportableOrderEntry) orderEntry).setScope(scope);
-            }
+      AppEngineStandardMavenLibrary.getLibraryByMavenDisplayName(addedLibrary.getName()).ifPresent(
```

Comment:
Oh yea - I missed that part because of the refactoring.

Could you restore it to the format I had it to prevent such deep nesting? I think it makes it easier to read.